main.js

```diff
@@ -0,0 +1,65 @@
+var text = document.getElementsByTagName('input')[0];
+var add = document.getElementById('add_todo');
+var list = document.getElementsByTagName('ul')[0];
+var todoQuant = 0;
+
+
+//create TodoList prototype
+var TodoList = function(listID, name, taskList) {
+  this.id = listID;
+  this.listName = name;
+  this.tasks = [];
+  this.createTask = function (taskID, task, completed) {
+    var created = {
+      id: taskID,
+      task: task,
+      completed: false
+    };
+    return created;
```

Comment:
This example works great, but you could also create a new object constructor for a Todo.